target/linux/ath79/image/generic-ubnt.mk

```diff
@@ -73,6 +95,18 @@ define Device/ubnt_nano-m
 endef
 TARGET_DEVICES += ubnt_nano-m
 
+define Device/ubnt_nanostationacl
```

Comment:
`define Device/ubnt_nanostation-ac-loco` please.

This way you can get rid of SUPPORTED_DEVICES, as the correct image metadata will be set. The `nanostation-acl` isn't required in the SUPPORTED_DEVICES as we don't have such a board in ar71xx.

Reply:
Thanks for pointing that out. It was not entirely clear to me how those names are related to each other.